registry/storage/driver/s3-aws/s3_test.go

```diff
@@ -76,6 +77,14 @@ func init() {
 			}
 		}
 
+		virtualHostedStyleBool := true
+		if secure != "" {
```

Comment:
copy/paste issue - should be:
```suggestion
		if virtualHostedStyle != "" {
```

Reply:
updated in next push